javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -44,6 +44,7 @@
 import java.util.*;
 import static com.github.javaparser.ast.Node.Parsedness.*;
 import static java.util.Collections.unmodifiableList;
+import com.github.javaparser.ast.Node;
```

Comment:
It's a dumb generator :-) I removed it while cleaning up imports for last release, now a generator puts it back.